src/conditionalAndLoops.js

```diff
@@ -169,7 +243,17 @@ function timespanToHumanString(startDate, endDate) {
  *    365, 10 => '365'
  */
 function toNaryString(num, n) {
-	throw new Error("Not implemented");
+	return ""+DecToBASE(num,n)
```

Comment:
You can type cast to string in other ways too!
A bonus challenge: Will your implementation work for hexadecimal (base 16)?
Try to update your implementation to support all bases till 16.

Reply:
Done.
I made a function which intercepts the remainder and maps [10...15] to [A...F]

Reply:
Great job.